source/wp-content/themes/wporg-parent-2021/sass/blocks/_heading.scss

```diff
@@ -0,0 +1,5 @@
+.wp-block-heading {
```

Comment:
Separate files like this are intended to be added to the frontend too, so I've moved this code directly into `editor.scss`. The other files imported into `editor.scss`, `button` and `navigation`, are frontend styles, added to ensure the styling is correct across both.